env0/resource_custom_flow_assignment.go

```diff
@@ -0,0 +1,119 @@
+package env0
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func setCustomFlowAssignmentId(d *schema.ResourceData, assignment client.CustomFlowAssignment) {
+	d.SetId(fmt.Sprintf("%s|%s", assignment.ScopeId, assignment.BlueprintId))
+}
+
+func getCustomFlowAssignmentFromId(d *schema.ResourceData) client.CustomFlowAssignment {
+	id := d.Id()
+	split := strings.Split(id, "|")
+
+	var assignment client.CustomFlowAssignment
+	assignment.ScopeId = split[0]
+	if len(split) > 1 {
```

Comment:
shouldn't we throw an error if `len(split) != 2` ?

Reply:
Added an if/else.
Keeping `len(split) > 1` because may need it to be longer in the future (E.g. if more scope types are added).